pkgs/development/libraries/clutter/default.nix

```diff
@@ -19,12 +19,20 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ gtk3 ];
   nativeBuildInputs = [ pkg-config gobject-introspection ];
-  propagatedBuildInputs =
-    [ libX11 libGL libGLU libXext libXfixes libXdamage libXcomposite libXi cogl pango
-      atk json-glib gobject-introspection libxcb libinput libgudev libxkbcommon
-    ];
+  propagatedBuildInputs = [
+    cogl pango atk json-glib gobject-introspection
+  ] ++ lib.optionals stdenv.isLinux [
```

Comment:
I think it would be better to use the condition `(!stdenv.isDarwin)`,  because most of our (even more) obscure platforms use X11 in some form.

Reply:
Done. Thanks!